src/Sylius/Bundle/ResourceBundle/AbstractResourceBundle.php

```diff
@@ -97,9 +97,9 @@ protected function getDoctrineMappingDirectory(): string
      *
      * @return string
      */
-    protected function getModelNamespace(): ?string
+    protected function getModelNamespace(): string
```

Comment:
Changing the return typehint by removing nullability is a no-go zone :) It would break 3rd party applications if they're using the currently correct method signature with `?string` return typehint.